### PR TITLE
Delete button on asset view page

### PR DESCRIPTION
**Pull request by akemidx**
# Description

A button has been added to the Asset detail view page that allows for deletion (archiving) of that asset.

Additionally, the dropdown menu in the upper right has been removed and replaced with buttons on the right column.
Screenshot 2023-04-27 at 7 12 50 PM

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Tested Locally.

# Checklist:

- [x] I have read the Contributing documentation available here: https://snipe-it.readme.io/docs/contributing-overview
- [x] I have formatted this PR according to the project guidelines: https://snipe-it.readme.io/docs/contributing-overview#pull-request-guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by akemidx** [comment]
There is a placeholder comment on the hardware/view blade for a generic image if no image is provided by the user

File: resources/lang/en/admin/hardware/general.php [hunk 1]
```diff
@@ -12,6 +12,7 @@
     'clone'  					=> 'Clone Asset',
     'deployable'  				=> 'Deployable',
     'deleted'  					=> 'This asset has been deleted.',
+    'delete_confirm'            => 'Are you sure you want to delete this asset?',
```
Comment (snipe) [inline resources/lang/en/admin/hardware/general.php:15]: I don't think we need this? We already have a translation string for "Are you sure you want to delete <item>".

`{{ trans('general.sure_to_delete') }}`

I'd really ideally prefer that be something that takes a variable for better localization, but it would at least mimic what we already have on the table delete buttons.

File: resources/views/hardware/view.blade.php [hunk 2]
```diff
@@ -887,11 +882,13 @@
                                 @endif
 
                                 @if (($asset->assignedTo) && ($asset->deleted_at==''))
-                                    <h2>{{ trans('admin/hardware/form.checkedout_to') }}</h2>
+                                    <div style="text-align: left">
+                                        <h2>{{ trans('admin/hardware/form.checkedout_to') }}</h2>
                                         <p>
                                         @if($asset->checkedOutToUser()) <!-- Only users have avatars currently-->
-                                            <img src="{{ $asset->assignedTo->present()->gravatar() }}" class="user-image-inline" alt="{{ $asset->assignedTo->present()->fullName() }}">
```
Comment (snipe) [inline resources/views/hardware/view.blade.php:902]: Why was this class removed?

File: routes/web/hardware.php [hunk 3]
```diff
@@ -146,9 +146,9 @@ function () {
             [AssetFilesController::class, 'show']
         )->name('show/assetfile');
 
-        Route::delete('{assetId}/showfile/{fileId}/delete',
```
Comment (snipe) [inline routes/web/hardware.php]: This will break file deletion on assets. We already have a delete route - the one we use for table-button deletes. No route changes should be required here, unless I'm missing something (and I might be)

File: resources/lang/en/admin/hardware/general.php (hunk 1 above)
Comment (akemidx) [reply resources/lang/en/admin/hardware/general.php:15]: I can fix that in a bit or this weekend.

File: resources/views/hardware/view.blade.php (hunk 2 above)
Comment (akemidx) [reply resources/views/hardware/view.blade.php:902]: Oh that might have been accidentally removed when I was getting rid of the image I put in to test originally and just forgot that was supposed to be there.

**Comment by akemidx** [comment]
we can chat Monday. i think the merge i did messed some stuff up. I also just need to commit less i guess.

File: routes/web/hardware.php (hunk 3 above)
Comment (akemidx) [reply routes/web/hardware.php]: I can double check we still use this. With the rewrite it probably isn't used. but I can look at it this weekend.

Comment (snipe) [reply routes/web/hardware.php]: We don't work on weekends :) It will keep until monday :D

File: resources/views/hardware/view.blade.php (hunk 2 above)
Comment (akemidx) [reply resources/views/hardware/view.blade.php:902]: this might have been an accident when clearing out something else

File: routes/web/hardware.php (hunk 3 above)
Comment (akemidx) [reply routes/web/hardware.php]: that was old, and couldn't get it to work. but it should be gone off the final bit

File: resources/lang/en/general.php [hunk 4]
```diff
@@ -91,7 +91,6 @@
     'debug_warning'         => 'Warning!',
     'debug_warning_text'    => 'This application is running in production mode with debugging enabled. This can expose sensitive data if your application is accessible to the outside world. Disable debug mode by setting the <code>APP_DEBUG</code> value in your <code>.env</code> file to <code>false</code>.',
     'delete'  				=> 'Delete',
-    'delete_confirm'  		=> 'Are you sure you wish to delete :item?',
```
Comment (snipe) [inline resources/lang/en/general.php]: Why is this deleted?

File: routes/web/hardware.php [hunk 5]
```diff
@@ -146,9 +146,9 @@ function () {
...
-            [AssetFilesController::class, 'destroy']
-        )->name('delete/assetfile');
+        Route::delete('{assetId}/delete',
+            [AssetsController::class, 'destroy']
+        )->name('hardware/delete');
```
Comment (snipe) [inline routes/web/hardware.php]: This route already exists - we shouldn't have to add or remove anything from routes for this.

Comment (akemidx) [reply routes/web/hardware.php]: Oh, it was removed on my side and I had added it back. Sorry got crossed up.

File: routes/web/hardware.php (hunk 3 above)
Comment (snipe) [reply routes/web/hardware.php]: I'm still showing this route as deleted in the diff?

File: resources/lang/en/general.php (hunk 4 above)
Comment (akemidx) [reply resources/lang/en/general.php]: Oh, I thought I added this back in??

I originally thought we didn't need it cause you said we had a string already, but this was probably the strong you were talking about.

File: resources/lang/en/general.php [hunk 6]
```diff
@@ -264,7 +264,7 @@
     'accept_eula'           => 'Acceptance Agreement',
     'supplier'              => 'Supplier',
     'suppliers'  			=> 'Suppliers',
-    'sure_to_delete'    => 'Are you sure you wish to delete',
+    'sure_to_delete'    => 'Are you sure you wish to delete :item?',
```
Comment (snipe) [inline resources/lang/en/general.php]: We can't use a variable here without checking all of the other places we use this string, since we were not previously passing it an item.

Comment (akemidx) [reply resources/lang/en/general.php]: so this was fixed in my local? so i slapped a ? at the end and pushed the new commit

Comment (snipe) [reply resources/lang/en/general.php]: We don't actually want a ? there, since we tack on (via PHP and JS) the specific item name. Grep through the code for the places we use `sure_to_delete` as a translated string and you'll see what I mean. It would end up with "Are you sure you wish to delete? Asset #1234o8568?"

Comment (akemidx) [reply resources/lang/en/general.php]: oh this is why it loked the way it did when i pulled it up. yea easy fix.
